include/ibamr/IBFESurfaceMethod.h

```diff
@@ -472,14 +504,29 @@ class IBFESurfaceMethod : public IBStrategy
 
 protected:
     /*!
-     * Impose (pressure) jump conditions.
+     * Impose the jump conditions.
      */
     void imposeJumpConditions(const int f_data_idx,
-                              libMesh::PetscVector<double>& DP_ghost_vec,
+                              libMesh::PetscVector<double>& P_jump_ghost_vec,
+                              std::array<libMesh::PetscVector<double>*, NDIM>& DU_jump_ghost_vec,
                               libMesh::PetscVector<double>& X_ghost_vec,
                               const double data_time,
                               const unsigned int part);
-
+    /*!
+     * \brief Helper function for checking possible double-counting
+     *  intesection points
+     */
+    void checkDoubleCountingIntersection(int axis,
```

Comment:
If this function does not use any class state, then suggest moving it to `libmesh_utilities` as a static member function along with the other intersection routines.